package.json

```diff
@@ -51,15 +54,20 @@
     "karma-json-fixtures-preprocessor": "0.0.6",
     "karma-safari-launcher": "^1.0.0",
     "lint-staged": "^7.3.0",
-    "rollup-plugin-typescript2": "^0.4.1",
-    "rollup-stream": "^1.19.0",
     "ts-node": "^7.0.0",
     "tslint": "^5.11.0",
-    "vinyl-source-stream": "^2.0.0",
-    "watchify": "^3.11.0"
+    "typescript": "^3.2.4",
+    "uglifyjs-webpack-plugin": "^2.1.1",
+    "watchify": "^3.11.0",
+    "webpack": "^4.29.0",
+    "webpack-cli": "^3.2.1",
+    "webpack-merge": "^4.2.1"
   },
   "scripts": {
     "init": "npm install && npm install -g gulp && npm install -g gulp-cli",
+    "build": "gulp build && npm run build:bundle && npm run build:bundle:dev",
```

Comment:
> && npm run build:bundle && npm run build:bundle:dev [](start = 24, length = 52)

I don't think it makes sense to build both prod and dev for the default build command. A better layout here would be a separate build command from build:dev - also inlining the build:bundle calls, they are only one command. Once we separate build and build:dev they will be unnecessary